common/scripted_triggers/hfe_trigger_pirate.txt

```diff
@@ -98,7 +98,7 @@ hfe_trigger_pirate_raid_distance_check = {
 				limit = {
 					is_owned_by = prevprevprev
 				}
-				max_steps = 10
+				max_steps = 15
```

Comment:
Revert it

Reply:
There should be more than five Raiding Targets. Five is too few. I extended the maximum steps to compensate slightly for it.

Reply:
This is too heavy calculation.
I know what im doing

Reply:
Okay, done